### PR TITLE
Added empty recipients message

### PR DESCRIPTION
**Pull request by Olzie-12**
if a player is speaking in a radius based channel then they will receive a message if they're speaking to no-one

### CODE REVIEW

File: common/src/main/java/net/draycia/carbon/common/listeners/RadiusListener.java [hunk 1]
```diff
@@ -74,6 +76,10 @@ public RadiusListener(
                     return false;
                 });
             }
+
+            if (!event.recipients().isEmpty()) return;
```
Comment (jpenilla) [inline common/src/main/java/net/draycia/carbon/common/listeners/RadiusListener.java]: In general, I think our code style calls for always including brackets, not sure why it's not in the checkstyle. I think in this case
```java
if (condition) {
  // action...
}
```
would be better anyways though.

**Review by jpenilla — CHANGES_REQUESTED** [review]
I think it would be good to include a config option for this feature, should be able to go into the ConfigChatChannel

defaulting to true/enabled would be fine imo

**Comment by Olzie-12** [comment]
Okay no problem, I'll add it :)

**Comment by Olzie-12** [comment]
Ready to merge or make any changes if you'd like :)

**Pull request merged**